dom.bs

```diff
@@ -1,21 +1,12 @@
 <pre class='metadata'>
-Title: DOM Standard
+Group: WHATWG
 H1: DOM
 Shortname: dom
-Status: LS
-Group: WHATWG
-No Editor: true
-!Participate: <a href=https://github.com/whatwg/dom>GitHub whatwg/dom</a> (<a href=https://github.com/whatwg/dom/issues/new>new issue</a>, <a href=https://github.com/whatwg/dom/issues>open issues</a>, <a href=https://www.w3.org/Bugs/Public/buglist.cgi?component=DOM&amp;product=WebAppsWG&amp;resolution=--->legacy open bugs</a>)
-!Participate: <a href=https://wiki.whatwg.org/wiki/IRC>IRC: #whatwg on Freenode</a>
-!Commits: <a href=https://github.com/whatwg/dom/commits>GitHub whatwg/dom/commits</a>
-!Commits: [SNAPSHOT-LINK]
-!Commits: <a href=https://twitter.com/thedomstandard>@thedomstandard</a>
-!Tests: <a href=https://github.com/w3c/web-platform-tests/tree/master/dom>web-platform-tests dom/</a> (<a href=https://github.com/w3c/web-platform-tests/labels/dom>ongoing work</a>)
-!Translation (non-normative): <span title=Japanese><a href=https://triple-underscore.github.io/DOM4-ja.html lang=ja hreflang=ja rel=alternate>日本語</a></span>
-Logo: https://resources.whatwg.org/logo-dom.svg
+Text Macro: TWITTER domstandard
 Abstract: DOM defines a platform-neutral model for events, aborting activities, and node trees.
+Translation: ja https://triple-underscore.github.io/DOM4-ja.html
+!Participate: <a href=https://www.w3.org/Bugs/Public/buglist.cgi?component=DOM&amp;product=WebAppsWG&amp;resolution=--->legacy open bugs</a>
```

Comment:
Would this appear after Translation? I guess that's acceptable.

Reply:
Nope, the new metadata reorder-er takes care of it for us.

Reply:
Er, actually, it just gets dropped on the floor. https://github.com/tabatkins/bikeshed/issues/1161